src-vue/src/views/mods/LocalModsView.vue

```diff
@@ -14,6 +14,12 @@
                     </template>
                 </el-popconfirm>
                 {{ mod.name }}
+                <img
+                    v-if="mod.thunderstore_mod_string != null"
+                    src="/src/assets/thunderstore-icon.png"
+                    class="image"
+                    height="16"
```

Comment:
Probably not the right way to define size so I'm happy to hear what I should actually use ^^

Reply:
That's a good way to do it :)